### PR TITLE
Awesome Books: Features

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

## Updates:

*  Create a function to add books to the list.
* Create a function to remove books from the list.
* Use local storage to store the list.
* Create a function to display the list of books.

## Notes:

Coded by @afcastaneda223 and @smunozmo 

✅  - No linters errors detected. 
✅  - Used Javascript for functions. 
✅  - Book list created dynamically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Capstone
```
Comment (Temmarie) [inline README.md]: - [ ] Add the correct project name in the Readme file.
- [ ] Also, it'll be nice if you could add more information about the authors as well.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @smunozmo and @afcastaneda223     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
- [ ] Kindly use the correct linter file for the JavaScript module. It can be found [here](https://github.com/microverseinc/linters-config/tree/master/javascript).

- [ ] For your next milestone, be sure to use descriptive branch names, so that the reviewer can know what the milestone is about For example, `refactor-code-classes` . I hope this makes sense.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

File: README.md (hunk 1 above)
Comment (Davidosky007) [reply README.md]: done 👍

**Comment by Davidosky007** [comment]
Hi @smunozmo @afcastaneda223, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**